plugins/woocommerce/src/Blocks/BlockTypes/ProductImage.php

```
@@ -260,9 +260,14 @@ private function render_image( $product, $attributes, $image_id = null ) {
 			$attr['loading'] = $loading_attr;
 		}
 
-		$adjust_srcset = function ( $sources, $size_array, $image_src, $image_meta ) use ( $aspect_ratio ) {
+		$srcset_aspect_ratio = $aspect_ratio;
+		if ( is_string( $srcset_aspect_ratio ) && is_numeric( $srcset_aspect_ratio ) ) {
+			$srcset_aspect_ratio .= '/1';
+		}
+
+		$adjust_srcset = function ( $sources, $size_array, $image_src, $image_meta ) use ( $srcset_aspect_ratio ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$size_array'. (undefined)

(UnusedFormalParameter)